### ISSUE TITLE
Lack of accessibility

### ISSUE THREAD

**Issue by bingen13** [lapce/lapce#2746]
## Lapce Version
0.3.0 portable.

## System information

Windows 10 22H2 (AMD64) build 19045.3570

## Describe the bug

Executing the portable version of the program, the screen reader NVDA doesn't seem to be able to tell anything about the window, detect the widgets, read the text area, etc. There is absolutely no screen reader feedback so I cannot tell exactly what is going on or whether this is during a first-run install task or on normal operation.

## Additional information

NVDA version: 2023.3. If you wish to do testing, you can download it from https://www.nvaccess.org/download/
